schemas/schemas/questionOptionsv3.json

```diff
@@ -3,5 +3,14 @@
     "title": "v3 question options",
     "description": "Options for a v3 question.",
     "type": "object",
-    "properties": {}
+    "comment": {
+        "description": "Add the comment",
+        "type": "string"
+    },
```

Comment:
I think these lines are incorrect?

Reply:
Can you explain?  Also, is this questionOptionsv3 deprecated or is it somehow included from another schema?

Reply:
There shouldn't be a `"comment"` at the top level of the schema file (it doesn't do anything). The `"comment"` property should only be added under `"properties"`.